### ISSUE TITLE
[Feature-Name]:

### ISSUE THREAD

**Issue by Nyanteea** [department-of-veterans-affairs/va.gov-team#54765]
### Describe the problem

I am the primary researcher on the disability compensation experience team focusing on the 526 form. My team is in the process of determining KPIs and metrics for measuring refinements/changes to the form. While the DOMO dashboard is useful in understanding *where* bottlenecks are occurring, it would be great to have more contextual information about how. A number of 526 submissions are lost when the wrong information is entered into form fields despite directional messaging about appropriate inputs and error messages. Being able to see what users are entering into forms will allow us to determine when a problem is a backend problem, a UI/UX issue, or some combination of the two. Session replay software allows organizations to blur form fields and sensitive information and hide any unique IDs to prevent disclosure of PII/PHI. 

### Who will benefit

Every project team would benefit from the added layer of user feedback when finding pain points or assessing performance. 

### Describe your idea

Solution:  Session replay software like that offered by FullStory allows clients to view a sample of anonymous users visiting a site for a period of time at a specific point in the journey. It allows organizations/clients to blur form fields and sensitive information and hide any unique IDs to prevent disclosure of PII/PHI. 

### Provide evidence

Hundreds of 526 submissions are lost when the wrong information is entered into form fields despite directional messaging about appropriate inputs and error messages.

### Platform Mission

- [ ] Keeps these products and services available, stable, and secure.
- [X] Ensures they are consistent and meet high standards of quality in Veteran-centered user experience and technical approach.
- [X] Provides shared tools, software, support, and technical infrastructure to accelerate their development.
- [ ] Other

**Comment by jhouse-solvd** [comment]
@Nyanteea - Thanks so much for submitting this. This sounds like an opportunity for leveraging the Platform's Centralized Monitoring solution using Datadog. 

I've used FullStory and have a good sense of how it works. I can see how that type of tool would be a good fit for your need. I'd love to explore how we can offer this functionality to support your team (and others). 

Do you think [Datadog's Session Replay for Browser Apps feature](https://www.datadoghq.com/dg/real-user-monitoring/session-replay) could offer the features you're looking for?

**Comment by Nyanteea** [comment]
Yes, that tool would be great to have as an added layer of context.

**Comment by jhouse-solvd** [comment]
@Nyanteea - I wanted you to know that I'm inquiring with Datadog about the availability of this feature for our use. 

I'll follow up with you once I have more details. :) 

Thank you!

**Comment by Nyanteea** [comment]
thanks for the update!

**Comment by steel36** [comment]
Potential solution was discussed with submitter, using new Datadog capabilities.  Discussion was taken offline.  (need to ensure in the future that resolution is recorded in original ticket).

Original submitter is no longer available so resolution cannot be confirmed.
